packages/@sanity/form-builder/src/inputs/files/ImageInput/ImageInput.tsx

```diff
@@ -523,7 +522,7 @@ export default class ImageInput extends React.PureComponent<Props, ImageInputSta
       assetSources && assetSources?.length === 0 ? null : (
         <MenuItem
           icon={SearchIcon}
-          text="Browse"
```

Comment:
This change doesn't look relevant to the PR description. Was including it intentional.

Reply:
Sorry, I should have added a comment on it. Yes, it was intentional.
There was some feedback that the previous text was causing some confusion :)